### ISSUE TITLE
Include English and Hebrew typefaces on the Document Detail View and the Search Results 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#181]
Here are the designs for you to view: I have applied the typefaces that I'm proposing on [the document detail view ](https://www.figma.com/file/HpGBOZi9lO8B3nCAx3d4Fj/Princeton-Geniza-Project-team?node-id=922%3A957)and [the search results](https://www.figma.com/file/HpGBOZi9lO8B3nCAx3d4Fj/Princeton-Geniza-Project-team?node-id=922%3A2488). I will show the Arabic typefaces separately from this issue, would like to reach a decision on the English and Hebrew first.

Context: 
- After researching for typefaces, and conversing with the dev team we seem to agree on using the [Google Noto fonts](https://www.google.com/get/noto/), which include typefaces for Hebrew, English, and Arabic. 
- For this issue, I am using Noto Sans and Sans Serif, Noto Sans Hebrew, and Noto Sans Serif Hebrew. 
- On the document detail view pages I have proposed a variation with the serif and another with the sans serif font for the description and the transcription text. My sense is that the sans serif is easier to read (especially with this font) now that I'm looking at it, what do you think?
- Are numeric values in Hebrew represented by letters of the alphabet? (Because I was thinking none of the Hebrew typefaces seem to have numeric values and got scared!) @thatbudakguy @rlskoeser @mrustow @richmanrachel 

Note: this issue does not include display typeface (which might/could be used for the page title or pull quotes on content pages).

**Comment by gissoo** [comment]
@mrustow @richmanrachel would love to hear your thoughts here, and let me know what you think about the legibility of the typefaces applied to the pages I have included on this issue. Also, is the Hebrew portion legible?

**Comment by mrustow** [comment]
- We would much prefer a serif font. Noto serif is OK, but only at medium or even semibold. The regular and other lighter weights are too elongated (rectangular). The sans serif fonts in Hebrew are like 60s modernist architecture: jarring and ubiquitous in Israel, and less like the fonts our scribes used and we are used to in printed books.
- no strong preferences for Latin fonts. Do the Hebrew, Arabic and Latin fonts have to be the same? 
- Of the Google Hebrew fonts, the only one we like is Frank Ruhl.
- Numeric values in Hebrew are no longer (since the 19th c) letters of the alphabet; they are the same numbers European languages use.
- Marina has strong feelings about Arabic fonts too, whenever this becomes relevant.

**Comment by gissoo** [comment]
@mrustow thank you for writing! Your comments are super helpful! Your comment about numeric values is so fascinating – for some reason I couldn't find more explicit information to confirm that this is the case.

Re: "Do the Hebrew, Arabic and Latin fonts have to be the same?" They certainly don't have to be same – however this approach would increase the performance, its file size ends up being a lot smaller and it would lower the loading time on the site (@thatbudakguy may know more) – on the design side for our purposes this approach ensures a bit more harmony between the fonts we will use but also specifically with Noto each typeface has so many variations with weights which gives us a lot more flexibility with design. 

@mrustow after I read your comment I have been doing primary and secondary research differently on Hebrew typefaces – I'll keep you posted, hopefully I'll have some updates by our design review meeting on Tuesday – I would like to do more exploration.

**Comment by gissoo** [comment]
Spoke with Marina and Rachel on Thursday. Out of the proposals I made for Hebrew typefaces we agreed on using ["Taamey Frank CLM"](https://culmus.sourceforge.io/taamim/)! From [what I found](https://en.m.fontke.com/font/13732023/detail/) this is a free font but can always contact them to make sure. 

Regarding the Latin typeface, I'll be looking for a narrower typeface that is still readable. Marina thinks Lato is too wide in comparison with the Hebrew typeface.

I will be looking for a more
